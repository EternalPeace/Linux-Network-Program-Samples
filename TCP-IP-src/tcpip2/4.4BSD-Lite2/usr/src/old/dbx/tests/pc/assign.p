program assign (input, output);

var a : array [1..100] of char;
begin
    a := 'blah';
    write(output, a);
end.
