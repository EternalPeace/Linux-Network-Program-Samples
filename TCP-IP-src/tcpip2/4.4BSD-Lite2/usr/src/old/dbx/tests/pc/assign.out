reading symbolic information ...
[1] stop at 6
[1] stopped in program at line 6
    6       write(output, a);
'blah                                                                                                ' 
'xlah                                                                                                ' 
'xyzzy' 
xyzzyexecution completed
'xyzzy' 

